### ISSUE TITLE
Fanart setting in Music Wall broken?

### ISSUE THREAD

**Issue by sialivi** [pix/aeon#156]
Section: Music
View: Wall

Does the Skin Setting "Fanart" do anything? It doesn't seem to change anything for me. I get the same fanart no matter if it's enabled or disabled.

**Comment by phil65** [comment]
where do you mean exactly? library, playlist, files mode? i know that fanart is still missing in playlist, will add that.

**Comment by sialivi** [comment]
Library

**Comment by phil65** [comment]
artist, album or song level? fallback activated? fanart.jpg in album folder?

**Comment by sialivi** [comment]
1) Album
2) Tried both
3) No

**Comment by phil65** [comment]
is the artist fanart visible? or does it show the global music background?

**Comment by sialivi** [comment]
Global music

**Comment by phil65** [comment]
hmm working here....
this is the code:
      <control type="largeimage" id="102">
        <include>FullscreenDimensions</include>
        <visible>!Control.IsVisible(103) + [Control.IsVisible(57) | Control.IsVisible(50) | Control.IsVisible(551) | Control.IsVisible(556) | Control.IsVisible(53)] + !IsEmpty(ListItem.property(fanart_image)) + Skin.HasSetting(musicfanart)</visible>
        <fadetime>400</fadetime>
        <texture background="true">$INFO[ListItem.property(fanart_image)]</texture>
        <include>Animation_FanartFade</include>
      </control>
      <control type="largeimage" id="103">
        <include>FullscreenDimensions</include>
        <visible>[Control.IsVisible(57) | Control.IsVisible(50)| Control.IsVisible(551) | Control.IsVisible(556) | Control.IsVisible(53)] + !IsEmpty(Container.Property(fanart_image)) + Skin.HasSetting(musicfanart)</visible>
        <fadetime>400</fadetime>
        <texture background="true">$INFO[Container.Property(fanart_image)]</texture>
        <include>Animation_FanartFade</include>
      </control>
screenshots perhaps?
you´re using the master branch?

**Comment by sialivi** [comment]
Yes, it's the master branch. I'll grab a screenshot later today.

**Comment by sialivi** [comment]
I'm not sure what happened but today the button suddenly work. However, it does feels slightly unintuative:

Fanart (enabled): Shows artist fanart
Fanart (disabled): Shows slideshow of global music backgrounds

I would expect Fanart (disabled) to remove the background pictures, not just toggling between which background pictures to show.

Instead of a "Fanart" toggle button, would it be possible to have some sort of combobox or radiobutton called "Background" which allow you to switch between "none", "fanart" and "slideshow"? I think that would be more intuative.

**Comment by phil65** [comment]
that´s why we have a fallback option, perhaps we should make that option accessible from the media menu (?)

**Comment by sialivi** [comment]
But the fallback option doesn't allow you to chose to have no background though (unless you create a blank image). The fallback is a fallback for when no fanart is available, but I think there should be an option to have no image even if there is fanart available.

**Comment by phil65** [comment]
hmmm but why havin no background? I don´t want to clutter the skin with options..
apart from that, I think that in some time I´ll rework the background controls for all areas completely. Don´t know exactly what I will change then. Will have to see.

**Comment by sialivi** [comment]
In Music Wall 95% of the background is covered by album covers. In Movies Multiplex the info panel covers nearly all of the background as well. In these cases I like to have the option of not having a background image.
